README.md

```diff
@@ -63,66 +62,65 @@ This script sets up some boilerplate dotfiles and installs common tools.
-  * Installs the Phoenix Framework
-  * Installs the Vim plugins configured in the `.vimrc` file
-  * Changes your default shell to ZSH
+* Generates a new SSH key pair if you don't have one yet
+* Installs Brew and common Brew packages
+* Installs Node using [Volta](https://volta.sh/)
+* Installs [OH-MY-ZSH](https://ohmyz.sh/)
+* Changes your default shell to ZSH
+* Copies some configuration files (see below)
 
 ### Dotfiles
 
 The following dotfiles are set up:
-  * `.aliases` for command aliases
-  * `.agignore` for folders that the `ag` command can ignore
-  * `.ackrc` for folders that the `ack` command can ignore
-  * `.exports` for exports
-  * `.editorconfig` for editor defaults
-  * `.gitconfig` your default git config, complete with your name and email
-  * `.gitignore` a global ignore file for git
-  * `.tmux.conf` a basic setup for tmux
-  * `.vimrc` a basic Vim setup, including basic Vim plugins
-  * `.zshrc` a basic ZSH setup, including OH-MY-ZSH
-
-If you want to add more aliases, you can add your own `.aliases.local` file to
-your home directory. This will be automatically loaded when your terminal
-starts.
-
-The following `.local` files are possible:
-  * `.aliases.local` for your own aliases
-  * `.exports.local` for your own exports
-  * `.gitconfig.local` for your custom git config
-  * `.vimrc.bundles.local` put your custom Vim plugins in this file (`NeoBundle 'ctrlpvim/ctrlp.vim'`)
-  * `.vimrc.local` for all other Vim customization
-  * `.zshrc.oh-my-zsh.local` for all your OH-MY-ZSH customization, before it is loaded
-  * `.zshrc.local` for all other ZSH customization
 
+* `.agignore` for folders that the `ag` command can ignore
+* `.ackrc` for folders that the `ack` command can ignore
+* `.editorconfig` for editor defaults
+* `.gitconfig` your default git config, complete with your name and email
+* `.gitignore` a global ignore file for git
+
+To add your own customizations to gitconfig, create a `.gitconfig.local` file in your home directory.
+
+### ZSH Config Files
+
+The following custom zsh config files are set up (in ~/.oh-my-zsh/custom/):
+
+* `asdf.zsh` sets up asdf manager
+* `aliases.zsh` for command aliases
+* `exports.zsh` for shell exports
+* `plugins.zsh` for zsh plugins
+
+If you want to add more aliases, you can add your own `aliases.local.zsh` file to
+your `~/.oh-my-zsh/custom/` directory. This will be automatically loaded when your terminal
+starts. The filename does not matter. Any `*.zsh` file in this directory will be loaded.
 
 ### Brew
 
 During installation some common Brew packages are installed, it includes the
 following packages:
 
-  * `ack`
-  * `elixir`
-  * `git`
-  * `hub`
-  * `node`
-  * `nvm`
-  * `postgresql`
-  * `redis`
-  * `the_silver_searcher`
-  * `vim`
-  * `watchman`
-  * `yarn`
-  * `zsh`
+* `ack`
+* `asdf`
+* `coreutils`
+* `findutils`
+* `git`
+* `hub`
+* `libyaml`
+* `the_silver_searcher`
+* `tmux`
+* `tree`
+* `vim`
+* `volta`
+* `watchman`
+* `wget`
+* `zsh`
```

Comment:
Related to the comment above "... change your default shell to ZSH", is this still needed since MacOS uses `zsh` by default now?

Reply:
TIL 👍